brainstormforce/ultimate-addons-for-gutenberg
# Added deprecated render icon js and bgcolor not applying in editor issue

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/223d982b-85d7-4d9e-b153-41896d313c78?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Create an Info-box in master with the button.
- Switch to next-release and validation issue occurs
- Switch to `validation-and-other-issue` branch to see the issue is resolved or not.
- 2nd Drag and drop info-box in next-release enable button and add bg color check is it applying in the editor or not.
- If not it is fixed in `validation-and-other-issue` branch.

### Checklist:
